db/migrate/20210429200726_update_foreign_key_types.rb

```diff
@@ -0,0 +1,30 @@
+class UpdateForeignKeyTypes < ActiveRecord::Migration[6.1]
+  TABLE_COLUMNS = {
+    analytics: %i[jurisdiction_id],
+    audits: %i[auditable_id associated_id user_id],
+    close_contacts: %i[enrolled_id],
```

Comment:
I know that at least some jurisdictions use this differently but do any jurisdictions use the `assigned_user` to assign specific user ids? If so then we may want to change that column to bigint as well

Reply:
for now there's an upper limit of `999,999` for assigned user, so I don't think that would be the case?